README.md

```diff
@@ -1,5 +1,9 @@
 # Debug Edge
 A small bridge board to link a programmer to a PCB, via a card edge connector.
+See [debug-edge.io](https://debug-edge.io/) for more the full specification and rationale.
+
+![A 3d render of a completed DebugEdge PCB](DebugEdge.png)
```

Comment:
Could you please add a double space at the end of the line here, so it puts the next image after a line break?

Reply:
Done!